build.gradle

```diff
@@ -405,6 +405,18 @@ tasks.register('cmakeJniLib', Exec) {
     standardOutput = outputStream
 }
 
+// Makes sure that `./gradlew clean` removes all JNI build artifacts
+tasks.clean.doFirst {
```

Comment:
can we not just remove build folder here, since with your earlier change you moved all these files to the `build` folder

Reply:
I think `Release` is still out there, I can see if I can add it under the `build` folder

Reply:
@navneet1v should be good now on the paths, let me know.